bots/orders/OrderFormNotion.ts

```diff
@@ -39,57 +43,67 @@ export default class OrderForm {
             });
```

Comment:
I think we could add a last_edited_time filter for if it's not the first sync to only see changed orders. or if that doesn't work, you could first query the parent page's last_edited_time to see if it's changed since the last poll.

Reply:
that's a good idea actually, I think I'll add this into a new PR! would be especially good as the requisitions db size grows